config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleMetadataBuilder.java

```diff
@@ -55,6 +55,7 @@ public BundleMetadata build(final AnnotatedBundle annotatedBundle, final List<En
                     projectInfo.getGroupName(), projectInfo.getVersion());
             builder.description(annotatedEntity.getDescription());
             builder.environmentEntities(getEnvironmentDependenciesMetadata(dependentEntities));
+            builder.dependencies(annotatedBundle.getDependentBundles());
```

Comment:
@uppoju dependencies are only applicable to annotated bundle?

Reply:
yes

Reply:
so, If an unannotated bundle has dependency on other bundles, we will not show that in metadata?

Reply:
@graju256 , do we have to add dependencies for unannotated bundle case ?

Reply:
Done